### ISSUE TITLE
Scrollable region must have keyboard access

### ISSUE THREAD

**Issue by bsylwek** [joomla/joomla-cms#45683]
### Steps to reproduce the issue

How can I solve the following digital accessibility problem? This is very important now due to the new regulations from June 28 regarding the accessibility of websites?

Axe Dev Tools issues
Ensure elements that have scrollable content are accessible by keyboard
[more information Link opens in a new window](https://dequeuniversity.com/rules/axe/4.10/scrollable-region-focusable?application=AxeFirefox)
Element Location:

.before-collapse

<ul class="before-collapse _access-scrollbar">

To solve this problem, you need to fix at least (1) of the following:

    Element should have focusable content
    Element should be focusable

### Device and system info

* What version of Joomla were you using?

Joomla 4.4.13 and Cassiopeia theme

* What device were you using? (eg, phone, laptop)

* What web browser were you using? (eg, Chrome, Safari)

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by alikon** [comment]
Joonla 4.4  is in security mode only, that means, no new features or bugfixes will be accepted.: https://developer.joomla.org/roadmap.html

check on Joomla 5
